src/components/mobile/mobile.vue

```diff
@@ -3,26 +3,26 @@
     <template #header>
       <div
         class="
-          x-list x-list--horizontal x-list__item--expand x-list--padding-03 x-list--align-center
+          x-list x-list--horizontal x-list__item--expand x-list--padding-05 x-list--align-center
         "
       >
-        <Close class="x-button--ghost">
-          <ArrowLeftIcon />
+        <Close class="x-button--ghost x-padding--left-00">
+          <ArrowLeftIcon class="x-icon--l" />
         </Close>
         <SearchBox class="x-list__item--expand" />
       </div>
     </template>
 
     <template #sub-header>
-      <div v-if="$x.relatedTags.length > 0" class="x-list x-list--padding-03">
+      <div v-if="$x.relatedTags.length > 0" class="x-list x-list--padding-05 x-padding--top-00">
```

Comment:
replace the deprecated x-list--padding-XX by x-padding--XX

Reply:
add ` x-list__item--expand` class to avoid width issues on animation when select a RT